packages/frontend/src/library-mode-entrypoint.tsx

```diff
@@ -1,26 +1,25 @@
 import React from "react";
-import {
-    type Chain,
-    type ChainProviderFn,
-    Connector,
-    configureChains,
-    createConfig,
-} from "wagmi";
-import { ReadonlyConnector } from "./connectors";
+import { createConfig } from "wagmi";
+import {} from "wagmi/connectors";
+import {} from "@wagmi/connectors";
```

Comment:
Are these imports desired?

Reply:
Addressed in 2b08fa7bc414d239c464b9b99b118e73c47cb45f